### PR TITLE
Create ENAH-full-note

### PR DESCRIPTION
**Pull request by natch-nacht**
This is the full-note CSL file for the Escuela Nacional de Antropología e Historia in Mexico City, based on the new Editorial Standards of the school. They were developed to facilitate the clarity of critical apparatus in  publications (books, manuals, magazines and bulletins), theses and homework, using bibliographic managers such as Mendeley.

### CODE REVIEW

**Comment by adam3smith** [comment]
the file is missing the .csl extension. Edit it here: https://github.com/natch-nacht/styles/blob/patch-7/escuela-nacional-de-antropologia-e-historia-full-note to fix.

File: escuela-nacional-de-antropologia-e-historia-full-note [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+      <email>publicaciones.enah@inah.gob.mx</email>
+      <uri>http://www.enah.edu.mx/index.php/difu-cul/publicaciones</uri>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="anthropology"/>
+    <category field="history"/>
+    <summary>Estilo de citas de la Escuela Nacional de Antropología e Historia -- variante nota completa</summary>
+    <updated>2015-06-25T20:21:07-06:00</updated>
+    <!--ISSN revista Cuicuilco-->
+    <issn>1405-7778</issn>
```
Comment (adam3smith) [inline escuela-nacional-de-antropologia-e-historia-full-note]: what's up with the ISSN? If this isn't for that journal it shouldn't be included here.

Comment (natch-nacht) [inline escuela-nacional-de-antropologia-e-historia-full-note]: This CSL file is going to be used in all the publication types at school, including the journal.


--
Enviado desde mi iMac

**Comment by adam3smith** [comment]
there's an unused macro point-locators that you should delete. 
About the ISSN/journal -- does the journal really allow multiple different citations styles (i.e. full note, short note, author-date)

**Comment by adam3smith** [comment]
Great -- the only thing left is that we'd want to align the title with the id -- they should either both be nota-completa or both be full-note. @rmzelle has the final word on naming, so I'm waiting for him to chime in on this.

**Comment by rmzelle** [comment]
I'm actually fine with that. The title should become `<title>Escuela Nacional de Antropología e Historia (nota completa, Spanish - Mexico)</title>` or `<title>Escuela Nacional de Antropología e Historia (nota completa) (Spanish - Mexico)</title>`, though.

**Comment by natch-nacht** [comment]
I think <title>Escuela Nacional de Antropología e Historia (nota completa) (Spanish - Mexico)</title> is more appropriate,  and also the other as <title>Escuela Nacional de Antropología e Historia (nota corta) (Spanish - Mexico)</title>  and <title>Escuela Nacional de Antropología e Historia (autor-fecha) (Spanish - Mexico)</title>, respectively

**Comment by adam3smith** [comment]
great, go ahead and rename and we're ready to merge.

**Comment by natch-nacht** [comment]
Only on the title section or even short-title's

*P. Antrop. Fís. J. Ignacio Flores S.*

2015-06-30 16:44 GMT-05:00 Sebastian Karcher <notifications@github.com>:

**Comment by adam3smith** [comment]
no, leave short-title as is.

**Comment by natch-nacht** [comment]
Done

**Comment by rmzelle** [comment]
> About the ISSN/journal -- does the journal really allow multiple different citations styles (i.e. full note, short note, author-date)

@natch-nacht, I'm still interested in an answer to this question. If the journal only uses one style variant, we'd like to create a separate (dependent) CSL style for it.

**Comment by natch-nacht** [comment]
I asked the Publications chief and she replied me that Revista Cuicuilco (Cuicuilco Journal) only allows author-date variant of the ENAH style, and the rest of the publications (books, manuals, bulletins, etc.) may take note variants, but the Editorial Standars of the school are the same for the journal and other types of publications. 

With create a separate (dependent) CSL style you mean to duplicate the ENAH author-date and remove the ISSN from the code, to set it apart as revista-cuicuilco.csl? 


--
Enviado desde mi iMac

**Comment by adam3smith** [comment]
Basically yes. You'd just have to remove the ISSN from all of the ENAH styles (if you haven't already--I didn't check). A dependent style is actually not really a copy, it's more like a link with additional info -- you can see examples in the dependent folder in the repository. We can create that for revista-cuicuilco once the ENAH are up.

**Comment by natch-nacht** [comment]
Ok, thanks

--
Enviado desde mi iMac

**Comment by adam3smith** [comment]
this is looking good now. Thanks!

**Comment by rmzelle** [comment]
E.g. https://github.com/citation-style-language/styles/blob/master/dependent/nature-biotechnology.csl is a dependent style that links to nature.csl.

**Pull request merged**